planner/planner_APIs.go

```diff
@@ -30,6 +30,8 @@ const (
 	PhotoApiBaseURL    = "https://maps.googleapis.com/maps/api/place/photo?maxwidth=400&photo_reference=%s&key=%s"
 )
 
+var geocodes map[string]string
```

Comment:
I am curious what is the variable scope of `geocodes`. Is it a global variable for server application?

Reply:
It is a global variable.